### DISCUSSION TITLE
why xdp.c modified doesnt show error?

### DISCUSSION THREAD

**Discussion by kolinfluence** [cilium/ebpf#877] [Q&A] [answered]
i did "go build ." and there's no error message to errors i purposely made on xdp.c
how do i show compile error for xdp.c? or no matter what i do with xdp.c, it will not be reflected at the end of the "xdp" program output?
with ref to:
https://github.com/cilium/ebpf/tree/master/examples/xdp

**Comment by ti-mo** [comment, marked as answer]
Please see my reply at https://github.com/cilium/ebpf/discussions/876#discussioncomment-4341551. `go build` does not equal `go generate`.
